app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,19 @@
     <%= javascript_pack_tag 'application' %>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,700&display=swap" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <% if request.path.include?('providers/') %>
+      <%# TODO: The title below should include the provider's first name #%>
+      <meta property="og:title" content="Request for support from a provider -- Feed a Doc" />
```

Comment:
I could use some Rails expertise here. I'd like to get the id out of the `request.path`, find the provider with that id and display their name. If the provider is not found for some reason, it should probably safely fallback.

Reply:
Can I use AR in the view? Is there some existing mechanism for doing this sort of thing?

Reply:
Andrew helped me resolve this :raised_hands: